lib/apiservers/portlayer/swagger.yml

```diff
@@ -193,6 +193,48 @@ paths:
           description: "error"
           schema:
             $ref: "#/definitions/Error"
+  /storage/volumes/{name}:
+    get:
+      description: "Get a Volume Handle"
+      operationId: GetVolume
+      tags: ["storage"]
+      consumes:
+        - application/json
+        - application/octet-stream
+      produces:
+        - application/json
+      parameters:
+        - name: name
+          required: true
+          in: path
+          type: string
+      responses:
+        '404':
+          description: "Volume not found"
+          schema:
+            $ref: "#/definitions/Error"
+        '200':
+          description: "OK"
+          schema:
+            type: string
+    delete:
+      summary: "Remove a volume"
+      tags: ["storage"]
+      operationId: RemoveVolume
+      parameters:
+        - name: name
+          type: string
+          in: path
+          required: true
+      responses:
+        '200':
+          description: "Volume successfully removed"
+        '404':
+          description: "Volume not found"
```

Comment:
Hey, sorry for commenting after you got 3 lgtm, but do you really never forsee passing back any error message with your status code (e.g. schema $ref: "#/definitions/Error")?